src/cds/src/main/kotlin/org/icpclive/cds/cats/CATSDataSource.kt

```diff
@@ -0,0 +1,222 @@
+
+    @Serializable
+    @SerialName("broadcast")
+    @Suppress("unused")
+    data class Broadcast(
+        val text: String
+    ) : Run()
+
+    // NOTICE: May it
+    @Serializable
+    @SerialName("c.question")
+    @Suppress("unused")
+    data class Question(
+        val text: String
+    ) : Run()
+
+    @Serializable
+    @SerialName("contest")
+    @Suppress("unused")
+    data class ContestStart(
+        val contest_start: Int
+    ) : Run()
+
+    private val authLoader =
+        jsonLoaderService<Auth> { "$url/?f=login&login=$login&passwd=$password&json=1" }
+    private val problemsLoader =
+        jsonLoaderService<Problems> { "$url/problems?cid=${properties.getProperty("cid")}&sid=${sid()}&json=1" }
+    private val usersLoader =
+        jsonLoaderService<Users> { "$url/users?cid=${properties.getProperty("cid")}&sid=${sid()}&rows=1000&json=1" }
+    private val contestLoader =
+        jsonLoaderService<Contest> { "$url/contest_params?cid=${properties.getProperty("cid")}&sid=${sid()}&json=1" }
+    private val runsLoader =
+        jsonLoaderService<List<Run>> { "$url/console?cid=${properties.getProperty("cid")}&sid=${sid()}&rows=1000&json=1" }
+
+    private fun getDateTime(dateTime: String, formatter: DateTimeFormatter) = LocalDateTime
+        .parse(dateTime, formatter)
+        .toKotlinLocalDateTime()
+        .toInstant(TimeZone.of(timezone))
+
+    override suspend fun loadOnce(): ContestParseResult {
+        return parseAndUpdateStandings(
+            problemsLoader.loadOnce(),
+            usersLoader.loadOnce(),
+            contestLoader.loadOnce(),
+            runsLoader.loadOnce()
+        )
+    }
+
+    private fun parseAndUpdateStandings(
+        problems: Problems,
+        users: Users,
+        contest: Contest,
+        runs: List<Run>
+    ): ContestParseResult {
+        val problemsList: List<ProblemInfo> = problems.problems
+            .asSequence()
+            .mapIndexed { index, problem ->
+                ProblemInfo(
+                    letter = problem.code,
+                    name = problem.name,
+                    color = Color(0x000000),
+                    id = problem.id,
+                    ordinal = index,
+                    minScore = if (resultType == ContestResultType.IOI) 0.0 else null,
+                    maxScore = if (resultType == ContestResultType.IOI) problem.max_points else null,
+                    scoreMergeMode = if (resultType == ContestResultType.IOI) ScoreMergeMode.MAX_PER_GROUP else null
+                )
+            }
+            .toList()
+
+        val teamList: List<TeamInfo> = users.users
+            .asSequence()
+            .filter { team -> team.role == "in_contest" }
+            .map { team ->
+                TeamInfo(
+                    id = team.account_id,
+                    name = team.name,
+                    shortName = team.name,
+                    contestSystemId = StringBuilder().append(team.account_id).toString(),
```

Comment:
What is difference with `team.account_it.toString()`

Reply:
because team.account_id is Int